src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,249 @@
+  /*
+   * The following const statements
+   * make the required contexts which contain
+   * the state and methods available to the component
+   */
+
+  const {
+    analysisDetailsContext,
+    analysisDetailsContextUpdateSubmissionPriority,
+    analysisDetailsContextUpdateEmailPipelineResult
+  } = useContext(AnalysisDetailsContext);
+
+  const { analysisContext, analysisContextUpdateSubmissionName } = useContext(
+    AnalysisContext
+  );
+
+  // List of analysis details
+  const analysisDetails = [
+    {
+      title: getI18N("AnalysisDetails.id"),
+      desc: analysisContext.analysis.identifier
+    },
+    {
+      title: getI18N("AnalysisDetails.pipeline"),
+      desc: `${analysisDetailsContext.workflowName} (${analysisDetailsContext.version})`
+    },
+    {
+      title: getI18N("AnalysisDetails.priority"),
+      desc: analysisDetailsContext.priority
+    },
+    {
+      title: getI18N("AnalysisDetails.created"),
+      desc: formatDate({ date: analysisDetailsContext.createdDate })
+    },
+    {
+      title: getI18N("AnalysisDetails.duration"),
+      desc: getHumanizedDuration({ date: analysisDetailsContext.duration })
+    }
+  ];
+
+  /*
+        On change of checkbox to receive/not receive an email upon
+        pipeline completion update emailPipelineResult field
+    */
+  function updateEmailPipelineResult(e) {
+    analysisDetailsContextUpdateEmailPipelineResult(e.target.checked);
+  }
+
+  // Update analysis name
+  function updateSubmissionName(newSubmissionName) {
+    if (
+      newSubmissionName !== "" &&
+      newSubmissionName !== analysisContext.analysisName
+    ) {
+      analysisContextUpdateSubmissionName(newSubmissionName);
+    }
+  }
+
+  // Update analysis priority
+  function updateAnalysisPriority(updatedPriority) {
+    analysisDetailsContextUpdateSubmissionPriority(updatedPriority);
+  }
+
+  // Render priorities in priority dropdown (Admins only)
+  function renderPriorities() {
+    const priorityList = [];
+
+    for (let priority of analysisDetailsContext.priorities) {
+      priorityList.push(
+        <Select.Option key={priority} value={priority}>
+          {priority}
+        </Select.Option>
+      );
+    }
+    return priorityList;
+  }
+
+  /*
+   * The following renders the analysis details, and tabs
+   * for Samples, Share Results, and Delete Analysis which
+   * the components are only loaded if the corresponding
+   * tab is clicked
+   */
+  return (
+    <>
+      <Tabs
+        defaultActiveKey="analysis_details"
+        tabPosition="left"
+        style={{ marginLeft: 50, paddingTop: 25 }}
+        animated={false}
+      >
+        <TabPane
+          tab={getI18N("AnalysisDetails.details")}
+          key="analysis_details"
+          style={{ minWidth: 300 }}
+        >
+          <Col span={12}>
+            <Title level={2}>{getI18N("AnalysisDetails.details")}</Title>
+
+            <Row>
+              <Col xs={{ span: 12, offset: 0 }} lg={{ span: 12, offset: 0 }}>
```

Comment:
You don't need any of the `Row` and `Col` . You are already in block level stuff.  And the spacing you are using is actually causing issues with rendering long Analysis names.

Reply:
Fixed in 72d637b21235727da79541ddd7c8705162c4f535